src/streams/readStream.ts

```diff
@@ -91,6 +91,12 @@ Client.prototype.readStream = function <
       break;
     }
     default: {
+      const streamRevision:bigint = fromRevision;
+
+      if (streamRevision < 0) {
+        throw new InvalidArgumentError("fromRevision value must be a ulong value.");
```

Comment:
As there isn't a concept of `ulong` in js, I thing it would be better to avoid the dotnet-ism.

Possibly:

`"fromRevision value must be a non-negative integer."`

Reply:
Should we also check the max value? (I think 18446744073709551615)

Reply:
Sure George. I will make the changes.